lib/systemFlake.nix

```diff
@@ -244,7 +234,7 @@ mergeAny otherArguments (
               __dontExport = true; # in case user uses overlaysFromChannelsExporter, doesn't hurt for others
               inherit srcs;
             })
-          ] ++ sharedOverlays ++ (if (value ? overlaysBuilder) then (value.overlaysBuilder pkgs) else [ ]) ++ [ fupOverlay ];
+          ] ++ sharedOverlays ++ (if (value ? overlaysBuilder) then (value.overlaysBuilder pkgs) else [ ]) ++ [ flake-utils-plus.overlay ];
```

Comment:
I have a sense that it would be better to let the user include `flake-utils-plus.overlay`.

The overlays and tools might eventually grow over time, so to preserve the minimal character of `fup` I opine this should rather not be added by default.

Ignore me, if you don't agree.  :smile:

Reply:
`(flake-utils-plus.replOverlay self)` &mdash; according to the other comment.

Reply:
The package name will not overwrite anything and I'd really rather leave as little as possible to the user to do thus this changeset stays